File: app/src/main/java/com/shevelev/comics_viewer/activities/comics_creation/drag_lists/IListItemDragCreatorImages.java
package com.shevelev.comics_viewer.activities.comics_creation.drag_lists;

import android.graphics.drawable.Drawable;

/**
 * Interface to working with images
 */
public interface IListItemDragCreatorImages
{
    Drawable getPageImage(String fullBitmapFileName);

    Drawable getStubPageImage();
}

File: app/src/main/java/com/shevelev/comics_viewer/activities/comics_creation/drag_lists/IListItemDragCreatorImages.kt
package com.shevelev.comics_viewer.activities.comics_creation.drag_lists

import android.graphics.drawable.Drawable

/**
 * Interface to working with images
 */
interface IListItemDragCreatorImages {
    val stubImage: Drawable

    fun getPageImage(fullBitmapFileName: String?): Drawable?
}